osf-model/src/main/resources/ontology/osf-model.rdf

```diff
@@ -964,6 +1023,9 @@
        </owl:equivalentClass>  
     </owl:Class>
   
-    <!-- TODO: Comments that may appear on a Project (or other node categories?) are
-        not represented in this ontology -->
+    <!-- http://www.dataconservancy.org/osf-business-object-model#Comment
+         TODO: Comments that may appear on a node or on other comments -->
+    <owl:Class rdf:about="&osfbom;Comment">
```

Comment:
Sub-class `OSFBusinessObject`? It seems that a comment does have an ID assigned to it by the OSF UI.